core/template_parser.go

```diff
@@ -48,16 +48,18 @@ func ParseTemplate(versionedHeader *VersionedHeaderTemplate, style CommentStyle)
 	if err != nil {
 		return nil, err
 	}
-	return &templateResult{
-		actualContent:  builder.String(),
-		detectionRegex: regexp.MustCompile(regex),
+	return &ParsedTemplate{
+		ActualContent:  builder.String(),
+		DetectionRegex: regexp.MustCompile(regex),
 	}, nil
 }
 
-// injects reserved parameter into template data map
-// the template will be parsed a second time, file by file, with the computed .Year value
+// injects reserved parameter into template data map by setting values as template placeholders
+// the template will be parsed a second time, file by file, with the actual values
 func injectReservedYearParameter(currentData map[string]string) map[string]string {
-	currentData["Year"] = "{{.Year}}"
+	currentData["Year"] = "{{.StartYear}}-{{.EndYear}}" // backwards compatibility
```

Comment:
in retrospect... not a good idea. it should remain `currentData[Year] = {{.Year}}` and that will remain the only way to normalize date range (from `"2019-2019"` to just `"2019"`)